src/App.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-import { connect } from 'react-redux';
-
-import NavigationContainer from './nav/NavigationContainer';
-
-const App = (props) =>
-  <NavigationContainer />;
-
-const mapStateToProps = (state) => ({
-  isLoggedIn: state.app.isLoggedIn,
```

Comment:
I think the right approach here is to probably check for errors in each async action (like `fetchSubscriptions` etc..) and dispatch an `ACCOUNT_INVALID` error when we discover an incorrect api key which sends the user back to log in